packages/framework/block-std/package.json

```diff
@@ -35,17 +35,16 @@
     "@blocksuite/store": "workspace:*"
   },
   "exports": {
-    ".": "./src/index.ts"
+    ".": "./src/index.ts",
+    "./edgeless": "./src/edgeless/index.ts"
   },
   "publishConfig": {
     "access": "public",
     "main": "dist/index.js",
     "types": "dist/index.d.ts",
     "exports": {
-      ".": {
-        "module": "./dist/index.js",
-        "import": "./dist/index.js"
-      }
+      ".": "./dist/index.js",
```

Comment:
You removed the `types` property here, will make users won't be able to find the type declaration when they use this package with typescript `bundler` module resolution.

Reply:
Please check: https://github.com/Milkdown/milkdown/blob/main/packages/milkdown/package.json#L68

Reply:
I think this was missing in current codebase, just added to keep the config consistent.